Sources/TSCBasic/Path.swift

```diff
@@ -457,6 +458,79 @@ private struct WindowsPath: Path, Sendable {
         return !path.withCString(encodedAs: UTF16.self, PathIsRelativeW)
     }
 
+    /// When this function returns successfully, the same path string will have had the prefix removed,
+    /// if the prefix was present. If no prefix was present, the string will be unchanged.
+    static func stripPrefix(_ path: String) -> String {
+        return path.withCString(encodedAs: UTF16.self) { cStringPtr in
+            let mutableCStringPtr = UnsafeMutablePointer(mutating: cStringPtr)
```

Comment:
This looks unsafe, you're taking a mutable view onto an immutable pointer; I think you need to use withUnsafeTemporaryAllocation and copy to a new buffer first.